### PR TITLE
Add the client registration activity, edit the style of UI.

### PR DESCRIPTION
**Pull request by Enkhzul001**
I added client registration activity, and changed some of the style. I also added back arrow.

### CODE REVIEW

File: app/src/main/java/com/javahelp/frontend/MainActivity.java [hunk 1]
```diff
@@ -48,7 +48,15 @@ public void onClick(View view) {
             }
         });
 
-
+        // link the button to the second activity(front page)
+        buttoncr.setOnClickListener(new View.OnClickListener() {
+            @Override
+            public void onClick(View view) {
+                Intent intent = null;
```
Comment (Lam207) [inline app/src/main/java/com/javahelp/frontend/MainActivity.java:55]: Could these two be combined into one step?

Comment (Lam207) [reply app/src/main/java/com/javahelp/frontend/MainActivity.java:55]: The one above and below this message. Sorry this splits them apart. 
Like  `Intent intent = new Intent(MainActivity.this,ClientRegistrationActivity.class);`. Is there any specific reason behind separating these two?

Comment (Enkhzul001) [reply app/src/main/java/com/javahelp/frontend/MainActivity.java:55]: I am bit confused. Can you explain more?

Comment (Enkhzul001) [reply app/src/main/java/com/javahelp/frontend/MainActivity.java:55]: Ah wait, I see what you mean. I can combine them.

Comment (yuanshu777) [reply app/src/main/java/com/javahelp/frontend/MainActivity.java]: I think they mean the same, and I think both ways are ok

**Review by yuanshu777 — APPROVED** [review]
It looks good to me.
Thanks for adjust the style

**Review by Lam207 — APPROVED** [review]
Looks great!

**Review by BullDF — COMMENTED** [review]
I am not really familiar with the UI part so I am not sure where the specific codes are but I have three things in mind now:

1. Should we add a Verify Password field for the user to enter their password again?
2. Is it possible to set the `Gender` field to be a dropdown menu or multiple-choice in align with the `Gender` class we just added? We can have the text entry box for the "other gender" option if we decided to have that freedom. I just think that it will be very difficult to record that into the database if the user can enter whatever they want.
3. Probably rearrange the entry boxes? Would it make sense to enter the email and password first before personal information? I think this is what the majority of website do.

Thanks for the work Zula!

**Review by Lenzs — APPROVED** [review]
Everything looks great!

**Pull request merged**